### ISSUE TITLE
`experimentalProxyless` option breaks the selectors from the `@testing-library/testcafe`

### ISSUE THREAD

**Issue by Roy412** [DevExpress/testcafe#7439]
### What is your Scenario?

Enabling the `experimentalProxyless` option breaks any selectors from the `@testing-library/testcafe`.

### What is the Current behavior?

Just created a simple project based on the NextJS and passed `experimentalProxyless: true` to the testcafe options. And I get the following error from the simple test.
```Error: Cannot call method 'queryByRole' of undefined```

### What is the Expected behavior?

Selectors from `@testing-library/testcafe` should work `experimentalProxyless` option.

### What is your public website URL? (or attach your complete example)

https://github.com/Roy412/msw-testcafe-error/tree/testing_library_compatibility

### What is your TestCafe test code?

```
import { screen } from "@testing-library/testcafe";

fixture("Rest").page`http://localhost:3000/`;

test("should render the page correctly", async (t) => {
  await t
    .expect(screen.findByRole('heading',{ name: 'Loading...' }).exists).ok('Page loaded')
});
```

### Screenshots

<img width="891" alt="Screenshot 2022-12-23 at 6 14 57 AM" src="https://user-images.githubusercontent.com/34854380/209334801-9a3b25fb-4c48-49a8-8c8f-a5cce3a4db20.png">

### Steps to Reproduce

1. Clone the repo: https://github.com/Roy412/msw-testcafe-error/tree/testing_library_compatibility
2. Yarn install: `yarn`
3. Run testcafe test: `yarn testcafe`

### TestCafe version

2.2.0-rc.1

### Node.js version

v16.16.0

### Command-line arguments

NEXT_PUBLIC_IS_TESTCAFE=1 node runtestCafe.js

**Comment by miherlosev** [comment]
Hi @Roy412,

I can't reproduce the issue. I get an error during the `yarn testcafe` command's execution.
<img width="899" alt="Screenshot 2022-12-26 at 9 28 50 AM" src="https://user-images.githubusercontent.com/4133518/209512385-f3068192-0fd1-4013-927a-cd48f5086284.png">

Please fix the shared example.

**Comment by Roy412** [comment]
Hi @miherlosev 

I just fixed the issue & updated the repository. Sorry, I had to double-check it.

**Comment by miherlosev** [comment]
Hi @Roy412,

Thank you. I've reproduced the issue.

**Comment by miherlosev** [comment]
Fixed in `testcafe@2.3.1-alpha.1`.
